uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -169,8 +160,34 @@ define([
          *
          * @return {Promise} resolved with {Boolean}
          */
-        isSentBack: function() {
+        isSentBack: function () {
           return checkLeaveStatus.call(this, 'more_information_requested');
+        },
+
+        /**
+         * Check the role of a given contact in relationship to the leave request.
+         *
+         * @param {Object} contact - contact object
+         *
+         * @return {Promise} resolves with an {String} - owner/manager/none
+         */
+        roleOf: function (contact) {
+          var deferred = $q.defer();
+
+          if (this.contact_id === contact.id) {
+            deferred.resolve('owner');
+          } else {
+            LeaveRequestAPI.isManagedBy(this.id, contact.id)
+              .then(function (response) {
+                //TODO Implement check for Admin in MS5
+                if (response === true) {
+                  deferred.resolve('manager');
```

Comment:
lets keep this? We will add checks for Admin in future, so current way would support that.